src/fragments/lib/auth/js/enable-signin-signup-signout.mdx

```diff
@@ -100,12 +196,75 @@ async function confirmSignUp() {
   }
 }
 ```
+</Block>
+</BlockSwitcher>
+
+### Auto sign in after sign up
+
+If you enabled `autoSignIn`, the `sign up` function will dispatch `autoSignIn` hub event after successful confirmation.
+If authentication was successful, the event will contain `CognitoUser` in data object. If auto sign in failed, it will dispatch `autoSignIn_failure` event.
+
+```js
+import { Hub } from 'aws-amplify';
+
+function listenToAutoSignInEvent() {
+  Hub.listen('auth', ({ payload }) => {
+    const { event } = payload;
+    if (event === 'autoSignIn') {
+      const user = payload.data;
+      // assign user
+    } else if (event === 'autoSignIn_failure') {
+      // redirect to sign in page
+    }
+  })
+}
+
+```
+
+### Custom Attributes
+
+To create a custom attribute during your sign-up process, add it to the attributes field of the `signUp` method prepended with `custom:`.
+
+```js
+Auth.signUp({
+  username,
+  password,
+  attributes: {
+    email,
+    'custom:favorite_flavor': 'Cookie Dough'  // custom attribute, not standard
+  }
+})
+```
+
+Amazon Cognito does not dynamically create custom attributes on sign up. In order to use a custom attribute, the attribute must be first [created in the user pool](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html). To open the User Pool to create custom attributes using the Amplify ClI, run `amplify console auth`. If you are not using the Amplify CLI, you can view the user pool by visiting the AWS console and opening the Amazon Cognito dashboard.
```

Comment:
I'm not following this part. Since Cognito does not dynamically create custom attributes do I do this after creating them with the library above? If so I think we should add some directional text like "Now that you created the custom attributes you will also need to create them in Cognito..." Not sure if that is true but that is how it is reading for me.